examples/example01/wannier_gaas.py

```diff
@@ -131,8 +131,8 @@
     dict(
         position_cart=(1, 2, 0.5),
         radial=2,
-        ang_mtm_l=2,
-        ang_mtm_mr=5,
+        ang_mtm_l_list=2,
```

Comment:
This will conflict with #42 - you can either remove this change (it's already fixed there) or ok to keep and we'll take care of the merge conflict in #42